src/options/modules/CustomOptionTriggers.js

```diff
@@ -342,7 +342,16 @@ export async function registerTrigger() {
     AutomaticSettings.Trigger.registerSave("popupIconColored", applyPopupIconColor);
     AutomaticSettings.Trigger.registerSave("emojiPicker", updatePerLineStatus);
     AutomaticSettings.Trigger.registerSave("emojiPicker", updateEmojiPerLineMaxViaEmojiSize);
-    AutomaticSettings.Trigger.registerSave("emojiSearch", applyEmojiSearch);
+    // Thunderbird
+    if (typeof messenger !== "undefined") {
+        // document.getElementById("browser").style.display = "none";
```

Comment:
commented code? Either remove it ot use it?

Reply:
Yeah, this was an alternative to the below lines.  This line would completely hide the Omnibox options, while the below lines disable and gray them out. I was not sure what you would want to do...

![image](https://user-images.githubusercontent.com/15305150/172042574-0c163939-099c-4ae1-b1eb-94a9fe1fdce0.png)

Reply:
As the feature is not supported on Thunderbird, I would completely hide it.

Reply:
OK, I will uncomment this and remove the below lines.